README.md

```diff
@@ -1,32 +1,32 @@
 # dev-ops
 Install Ngninx and GH actions
 
-## Install Ngninx and frameworks
-
-* SSH onto the VM
-* `git clone https://github.com/dltxio/dev-ops.git && cd dev-ops && ./install.sh`
-
-## Deploy bot SSH PubKey
-
-GitHub secrets add the following key pairs.  SSH private key can be found in bitwarden as "Digital Ocean Deploy Bot Private Key".
-
-* DO_DEPLOY_HOST
-* DO_DEPLOY_USERNAME
-* DO_DEPLOY_KEY
-
-DO_DEPLOY_HOST:  The IP or DNS of the VM  
-DO_DEPLOY_USERNAME:  The user name on the VM that will be used in the SSH tunnel  
-DO_DEPLOY_KEY: The SSH private key  
-
+## Install Ngninx and frameworks on your Digital Ocean droplet
+1. Install the Digital Ocean Deploy Bot Private Key into your `~/.ssh` directory on your computer (found in BitWarden)
+2. SSH onto the VM
```

Comment:
Might be worth explaining how to do this - I had to be shown. Maybe everyone else knows though, but if you're not up on Linux it'd be good to explicitly show what to do

Reply:
I added the cmd syntax plus the instruction for adding the key is already there. Anything more can be googled